configs/application/config.json

```diff
@@ -1,15 +1,13 @@
 {
 	"comment": "Top-level application config for development, adding on top of the core config",
-	"mode": "",
-	"isAuthEnabled": false,
 	"storage": {
 		"topicToDataStoreAdapters" : {
 			"finsemble" : "LocalStorageAdapter",
 			"finsemble.workspace" : "LocalStorageAdapter",
 			"finsemble.workspace.cache" : "LocalStorageAdapter"
 		},
 		"dataStoreAdapters" : {
-			"LocalStorageAdapter": "$applicationRoot/adapters/LocalStorageAdapter.js"
+			"LocalStorageAdapter": "/adapters/LocalStorageAdapter.js"
```

Comment:
Should `$applicationRoot` be removed from this? Is this combined with applicationRoot when read from the configuration?